package.json

```diff
@@ -20,10 +20,10 @@
         "Blake Thompson <flippmoke>"
     ],
     "dependencies": {
-        "step"        : "~0.0.5",
-        "generic-pool": "~2.4.0",
-        "mapnik":       "3.5.14",
-        "mime"        : "~1.3.4",
+        "step"         : "~0.0.5",
+        "generic-pool" : "~2.4.0",
+        "@carto/mapnik": "~3.6.2-carto.0",
```

Comment:
At some point, we'll have to have the same versioning approach for `tilelive-*`, what do you think?

Reply:
Indeed, it is what I'm using now: https://github.com/CartoDB/tilelive-bridge/blob/2.5.1-cdb1/package.json#L2

Reply:
I meant, the way we are versioning: [`2.5.1-cdb1`](https://github.com/CartoDB/tilelive-bridge/blob/2.5.1-cdb1/package.json#L3) vs  ~3.6.2-carto.0

Reply:
Oh, I did not get it before. I thought you were talking about namespaces and so on. I have no strong opinion. Those were mostly due to the infrastructure we have now for packaging mapnik and node-mapnik.